.circleci/config.yml

```
@@ -331,6 +330,103 @@ jobs:
       - run:
           name: Check translation file for missing or empty entries
           command: ./tools/check_translations.sh
+  bump-version:
+    docker:
+      - image: cimg/python:3.9.16
+    resource_class: small
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Install requirements
+          command: sudo apt-get update && sudo apt-get install pcregrep
+      - run:
+          name: Enable virtual environment
+          command: echo "source .venv/bin/activate" >> $BASH_ENV
+      - run:
+          name: Request installation access token to authorize as Deliverino app
+          command: echo "export DELIVERINO_ACCESS_TOKEN=$(./.circleci/scripts/get_access_token.py)" >> $BASH_ENV
+      - run:
+          name: Config git repository to commit & push as Deliverino app
+          command: |
+            git config user.name "deliverino[bot]"
+            git config user.email "62934656+deliverino[bot]@users.noreply.github.com"
+            git remote set-url origin "https://x-access-token:$DELIVERINO_ACCESS_TOKEN@github.com/digitalfabrik/integreat-compass.git"
+      - run:
+          name: Bump version
+          command: bumpver update --tag=final
+      - run:
+          name: Get tag of newly created version
+          command: |
+            eval $(bumpver show --env -n)
+            echo "export CURRENT_VERSION=$CURRENT_VERSION" >> $BASH_ENV
+      - run:
+          name: Tag and push commit
+          command: |
+            git tag --annotate "${CURRENT_VERSION}"
+            git push origin --follow-tags "${CURRENT_VERSION}" HEAD
+      - run:
+          name: Merge version bump into develop
+          command: git checkout develop && git merge main --commit --no-edit && git push
+  create-release:
+    docker:
+      - image: cimg/python:3.9.16
+    resource_class: small
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Install requirements
+          command: sudo apt-get update && sudo apt-get install pcregrep
+      - run:
+          name: Enable virtual environment
+          command: echo "source .venv/bin/activate" >> $BASH_ENV
+      - run:
+          name: Request installation access token to authorize as Deliverino app
+          command: echo "export DELIVERINO_ACCESS_TOKEN=$(./.circleci/scripts/get_access_token.py)" >> $BASH_ENV
+      - run:
+          name: Get previous version tag
+          command: |
+            PREV_TAG=$(git describe --abbrev=0 --tags "${CIRCLE_TAG}^") || true
```

This will probably throw an error on the first release. I added this to the integreat workflow once there existed already a few tags, so I didn't care about this edge case. In this repo however, I assume this will cause the workflow to fail.

---

Before this is (eventually) merged into main we should do one manual release I guess.

---

Hmm, doing a manual release is quite cumbersome...
Maybe we could insert the initial commit as artificial prev_tag in case this fails? :thinking: 
```suggestion
            PREV_TAG=$(git describe --abbrev=0 --tags "${CIRCLE_TAG}^") || PREV_TAG="3dc00abb0799ecd9d97480594b58330ec77cf43e"
```